singer/schema.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, type=None, format=None, properties=None, items=None,
         self.patternProperties = patternProperties
 
     def __str__(self):
-        return json.dumps(self.to_dict())
+        return orjson.dumps(self.to_dict())
```

Comment:
Orjson returns back bytes, instead of string, so it would make sense to cast here to string before return.

Reply:
thanks, fixed by 5c272cfc82e75dec9d2e9b9b6b6086785cc90a1f